File: codekata/absolute_beginner/factorial.py
# factorial of a number
# get input as number from user
number=int(input())
# define factorial function
def factorial():
# initialize fact is equal to one
    fact=1
# range from 1 to number using for loop
    for i in range(1,number+1):
# factorial operation
      fact=fact*i
# return fact value
    return fact
# call factorial funtion and store it in output
output=factorial()
# print output value
print(output)

File: codekata/javascript/absolute_beginner/factorial.js
// factorial of a number
const readline = require('readline');
const inp = readline.createInterface({
  input: process.stdin
});
inp.on("line", (number) => {
  var fact=1;
	for(var i=1;i<=number;i++)
    {
      var fact=fact*i
      }
  console.log(fact);
});
